gradle/gradle-publish.gradle

```diff
@@ -0,0 +1,81 @@
+apply plugin: 'maven-publish'
+apply plugin: 'signing'
+
+
+ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID') ?: ''
+ext["signing.password"] = System.getenv('SIGNING_KEY_PASSWORD') ?: ''
+ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_KEY_FILE') ?: ''
+ext["SONATYPE_NEXUS_USERNAME"] = System.getenv('SONATYPE_NEXUS_USERNAME') ?: ''
+ext["SONATYPE_NEXUS_PASSWORD"] = System.getenv('SONATYPE_NEXUS_PASSWORD') ?: ''
+
+
+afterEvaluate {
+    android.libraryVariants.each { variant ->
+        def variantName = variant.name
+        def resolvedVersion = project.version
+        publishing.publications.create variantName, MavenPublication, {
+            println "$variantName resolvedVersion: ${resolvedVersion}  ${group} ${project.name}"
+            groupId group
+            version resolvedVersion
+            artifactId project.name
```

Comment:
Is `project.name` the module name here? So card and core can have their own artifact IDs?

Reply:
yes, `project.name` is the module name. So we upload  `com.paypal.android.core` and `com.paypal.android.card` for example